src/panels/logbook/ha-logbook.ts

```diff
@@ -217,13 +217,19 @@ class HaLogbook extends LitElement {
                   .datetime=${item.when}
                   capitalize
                 ></ha-relative-time>
-                ${item.domain === "automation" &&
-                item.context_id! in this.traceContexts
+                ${item.context_id! in this.traceContexts
```

Comment:
Shouldnt we check for script or automation domain?

```ts
["script", "automation"].includes(item.domain) &&
```

Reply:
it is easier to check the domain than to check all the `traceContexts`, so I would add it back